pkg/storage/s3/s3.go

```diff
@@ -45,11 +45,10 @@ type ObjectStorage struct {
 	// We must keep track of multi part uploads to s3, because the lib
```

Comment:
remove this comment now?

Reply:
so we don't need to track the multipart then?

Reply:
yes, we don't anymore :D , I'll remove it